package.json

```diff
@@ -29,7 +29,7 @@
     "deno"
   ],
   "engines": {
-    "vscode": "^1.60.0"
+    "vscode": "^1.77.0"
```

Comment:
Does this make a difference to the currently resolved version and is it essential? I'd like to avoid it because the TS plugin API communication has been brittle in the past

Reply:
Yes, this bit is essential. It's what is required to get `request.continuous` and the final argument in
```
new vscode.TestRunRequest(
              runData.request.include,
              runData.request.exclude,
              runData.request.profile,
              true,
            ),
```

Without that, we don't have the typings for the continuous run feature in vscode.